pyaedt/modeler/geometry_operators.py

```diff
@@ -1569,7 +1571,7 @@ def point_in_polygon(point, polygon):
             - ``1`` When the point is inside the polygon.
         """
         # fmt: off
-        tol = 1e-8
+        tol = tolerance
         if len(point) != 2:  # pragma: no cover
             raise ValueError("point must be a list in the form [x, y]")
```

Comment:
```suggestion
            raise ValueError("Point must be a list in the form [x, y].")
```
Start with a capital letter and conclude the complete sentence with a period.